src/app/static/src/js/components/admin/adminApp.vue

```diff
@@ -15,6 +15,12 @@
             <label class="font-weight-bold d-block">For individual users</label>
             <manage-user-permissions :all-users="users" @changed="getUsers"></manage-user-permissions>
         </div>
+        <div class="col-4">
```

Comment:
I'm assuming the settings are going to share this third column with the documents refresh UI.